test/info/info-help.test.js

```diff
@@ -19,10 +19,11 @@ describe('should print help for info command', () => {
 
     it('should respect the --color=false flag', () => {
         const { stdout, stderr } = runInfo(['help', '--color=false'], __dirname);
-        chalk.enabled = true;
-        chalk.level = 3;
-        const orange = chalk.keyword('orange');
-        expect(stdout).not.toContain(orange(usageText));
+        //unusual behavior for windows CI
+        if (process.platform !== 'win32') {
```

Comment:
We should respect `--color=false` when explicitly stated in windows too.
What issue is being faced here?

Reply:
I think `--color=false` is not working for windows.
![Screenshot at 2020-07-19 19-55-47](https://user-images.githubusercontent.com/46647141/87877106-03253b00-c9fa-11ea-8e40-a1f31a1289f4.png)

Unfortunately, I don't have a windows machine to test it locally :disappointed:

Reply:
Can you provide some additional info on how this `--color` flag works? I see it isn't defined anywhere?

Reply:
> I think --color=false is not working for windows.

I was asking why it isn't working :). Now we know it was removed somehow in some changes.
Reason I think it was not caught till now is `chalk` seems to be respecting [`NODE_DISABLE_COLOR`](https://nodejs.org/dist/latest-v12.x/docs/api/cli.html#cli_node_disable_colors_1) supplied by CI's environment and `colorette` is not.

> Can you provide some additional info on how this --color flag works? I see it isn't defined anywhere?

I wasn't aware it was removed from `cli-flags.js`. Here is the commit tree and flag description where you would find what it is supposed to do 😅️
https://github.com/webpack/webpack-cli/blob/530b4f76a45d7eb20648e9ee2981cbf4a707854d/lib/utils/cli-flags.js#L84:L89

Reply:
Dont know how but `webpack --help --color=false` works as expected in my local env.

Reply:
We don't have the flag better we should remove the tests too.

Reply:
> We don't have the flag better we should remove the tests too.

+1 from my side. /cc @webpack/cli-team for more opinions.

Reply:
+1 for remove, we can already tune stats color from core flag, no need here.
/cc @evilebottnawi

Reply:
Just clarify? On window it is not work out of box?